loleaflet/css/jquery-ui-lightness.css

```diff
@@ -881,7 +881,7 @@ button.ui-button::-moz-focus-inner {
 	font-size: 1.1em;
 }
 .ui-widget .ui-widget {
-	font-size: 1em;
+	font-size: 0.5em;
 }
```

Comment:
not the elements we are trying to target. Maybe they can then be removed (if they are not used anywhere else but for now and to make the things simple I would avoid changing this)